### ISSUE TITLE
dxvk-remix generates huge log files by spamming [RTX-Compatibility-Info] per frame

### ISSUE THREAD

**Issue by Kamilkampfwagen-II** [NVIDIAGameWorks/rtx-remix#243]
**Describe the bug**
dxvk-remix generated a LFS_d3d9.log file with the size of ~500MiB by spamming this line:
`info:  [RTX-Compatibility-Info] This application is pushing more unique buffers than is currently supported - some objects may not raytrace.`
The game respects the shader model override and it was set to 0 so no weird interactions with shaders were involved.

GPU: RTX 2060 Desktop
Driver: 536.67 (NVCleanInstall)
Game: [Live for Speed (free trial)](https://www.lfs.net/archive) v0.3H7
Wrappers used: DxWrapper

**Attach logs!**
I don't have the logs from that time.

**To Reproduce**
1. Install the game and configure [settings ](https://gist.github.com/Kamilkampfwagen-II/77440fa39bf79a0aa42de7440b4fbb3b).
2. Enter a race.
3. Play for few minutes.
4. See the huge `LFS_d3d9.log` file generated.

**Expected behavior**
A normal sized log file without the info spam written every frame.

**Comment by NV-DM** [comment]
REMIX-2063 for tracking.

**Comment by xbei-nv** [comment]
Is this still happening now? I just checked in some games and haven't seen large log file issues. Because we have wrapped the log with a macro which helps to write log only once into the log file when the cache size over the limit.

**Comment by NV-LL** [comment]
Since this ticket has been inactive for several months and we haven't been able to reproduce the problem internally, we're going to close this out. Please open another ticket if you continue to experience this issue.
